### PR TITLE
Feature/create negotiation pt1

### PR DESCRIPTION
**Pull request by EdwardNeuwirth**

### CODE REVIEW

File: client/src/components/AddMember/AddMember.js [hunk 1]
```diff
@@ -36,6 +36,16 @@ export default class AddMember extends Component {
...
 
+    const Checkbox = ({name}) => (
+      <input
+        name={name}
+        type="checkbox"
+        checked={permissions[name]}
+        value={permissions[name]}
+        onChange={this.handleCheckboxChange}
+      />
+    );
```
Comment (oliversisson) [inline client/src/components/AddMember/AddMember.js:47]: this isn't a general purpose checkbox, so maybe a name like `PermissionCheckbox`?

File: client/src/components/TitleAndDescriptionPage/TitleAndDescriptionPage.test.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    handleInputChange,
+    handleInputOnFocus,
+    handleInputOnBlur
+  };
+
+  props.handleInputChange = event => {
+    const { document } = props;
+    const { name, value } = event.target;
+    props.document = { ...document, [name]: value }
+  };
```
Comment (oliversisson) [inline client/src/components/TitleAndDescriptionPage/TitleAndDescriptionPage.test.js:22]: `props.handleInputChange` seems to be defined in two places in this test.

File: client/src/containers/CreateNegotiation/CreateNegotiation.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+      });
+    }
+  };
+
+  handleInputOnBlur = event => {
+    const { document } = this.state;
+    const { name } = event.target;
+    if (document[name] === '') {
+      this.setState({
+        document: { ...document, [name]: `${name}...` }
```
Comment (oliversisson) [inline client/src/containers/CreateNegotiation/CreateNegotiation.js]: can you not handle this with a placeholder?

File: client/src/containers/CreateNegotiation/CreateNegotiation.js [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+          <TitleAndDescriptionPage
+            document={document}
+            handleInputChange={this.handleInputChange}
+            handleInputOnFocus={this.handleInputOnFocus}
+            handleInputOnBlur={this.handleInputOnBlur}
+          />
+        </div>
+      );
+    }
+    return <div>default</div>;
```
Comment (oliversisson) [inline client/src/containers/CreateNegotiation/CreateNegotiation.js:41]: progressTracker - this is odd for me. can I ask you about it?

File: client/src/containers/CreateNegotiation/CreateNegotiation.js [hunk 5]
```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+import './CreateNegotiation.css';
+import TitleAndDescriptionPage from '../../components/TitleAndDescriptionPage';
```
Comment (oliversisson) [inline client/src/containers/CreateNegotiation/CreateNegotiation.js:3]: these two files feel very odd for me. either they should be combined, or some things on one page should be moved on the other, or something... can I chat with you and find out your reasoning for doing it this way?

**Review by oliversisson — COMMENTED** [review]
there are two odd things - can I ask you about them?

**Review by gaber85 — APPROVED** [review]
good stuff

**Pull request merged**